lib/member_page.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+require 'scraped'
+require 'date'
+require_relative './member_email_decorator'
+
+class MemberPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::AbsoluteUrls
+  decorator MemberEmailDecorator
+
+  field :id do
+    url.split('/').last
+  end
+
+  field :name do
+    noko.css('.page-header h2').text.tidy
+  end
+
+  field :image do
+    noko.at_css('.item-page img/@src').text
+  end
+
+  field :constituency do
+    table_field('Constituency').match("People's Representative for (.*)")[1]
+  end
+
+  field :birth_date do
+    Date.parse(table_field('Date of Birth')) rescue nil
+  end
+
+  field :gender do
+    table_field('Sex').downcase
+  end
+
+  field :email do
+    noko.xpath("//span[starts-with(@id, 'cloak')]").map(&:text).join(';')
+  end
+
+  field :cell do
+    noko.xpath("//tr/td[contains(., 'Mobile Phone')]").text
+        .match(/Mobile Phone[[:space:]]*?\:[[:space:]]*?([^\nH]+)/)
+        .to_a[1].to_s.tidy.gsub(' or', ';')
+  end
+
+  field :phone do
```

Comment:
(For some reason Github has lost my comment on this one twice now. Apologies if you're actually getting it multiple times!)

I think this field is too complex to not be tested. Both it and `cell` could also benefit greatly from comments giving examples of the the layout(s) for this to make it much easier for the person who needs to come along later and expand it for one that gets added in a subtly different format.